src/state/index.ts

```diff
@@ -2,6 +2,9 @@ import { connectRouter, RouterState } from 'connected-react-router';
 import { createBrowserHistory, History } from 'history';
 import { combineReducers, createStore, Reducer } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
+import { persistReducer, persistStore } from 'redux-persist';
+import autoMergeLevel1 from 'redux-persist/es/stateReconciler/autoMergeLevel1';
+import storage from 'redux-persist/es/storage';
```

Comment:
Same question: `import { storage } from 'redux-persist';` or `import { storage } from 'redux-persist/es';` would be better

Reply:
They don't export these members. The type declaration file is wonky at best. Feel free to take a stab at it.

Reply:
Because they use default exports I have imported it like so:

```ts
import { default as storage } from 'redux-persist/es/storage';
```